src/content/reference/manifest.mdx

```diff
@@ -610,6 +610,29 @@ volumes:
 
 Sets the working directory of your development container.
 
+### forward ([string], optional)
+
+You can also declare global forwards that will be used by all dev environments.
+
+As in `forward` in dev section, several formats are accepted for this, but it is necessary to indicate the name of the service to be forwarded using the notation `localPort:remoteService:remotePort`.
+
+```yaml
+forward:
```

Comment:
Let's document the different notations in different samples, like here:
https://www.okteto.com/docs/reference/manifest/#forward-string-optional